docs/components/form-associated.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Form-Associated Components
+sidebar_label: Form-Associated Components
+description: Form-Associated Stencil Components
+slug: /form-associated
+---
+
+# Building Form-Associated Components in Stencil
+
+As of $VERSION Stencil has experimental support for form-associated custom
```

Comment:
Since we normally put experimental features behind flags, should we s/experimental/initial?
```suggestion
As of $VERSION Stencil has initial support for form-associated custom
```

Reply:
yeah, that might be something to discuss a little bit - I think we floated this being an 'experimental' feature but I don't know if we made a solid decision. The advantage of doing so I guess is that we can change APIs more freely if we run into some problems with them but I'm not sure what exactly makes the most sense

Reply:
I was under the impression we were initially shipping this as experimental. The feature PR also states it as experimental

Reply:
yeah, it feels like we've landed on something that will work and that we'd only possibly add to later, i.e. adding some nicer API around validation or something like that (just spitballing not proposing haha). like what we have now feels very close to the spec and minimal, so I wouldn't anticipate we'd need to massively rework parts of it in the future and so an actual 'experimental' label wouldn't precisely be warranted

Reply:
To round this out, I think it makes sense to just say "external" here and to leave the `@Component` option and the decorator as-is for now

Reply:
> To round this out, I think it makes sense to just say "external" here and to leave the @Component option and the decorator as-is for now

I think I'm lost, what parts 'external'?

Reply:
ohh `s/external/initial` 🤪